### PR TITLE
fix: Prevent jump on node selection (no auto scroll to center)

### LINKED ISSUE #391: Disable Automatic Scrolling on Selection
I apologize in advance if this question has already been asked/request has already been made--I searched the issue tracker for anything like this and I was not able to find anything specifically targeting this issue, though it's related to #259 and #257 

**Is your feature request related to a problem? Please describe.**
I'm working on a project and we'd like to be able to disable the automatic centering of the most recently selected node. Right now, whenever we select a node, the scrollTop is set due to:
https://github.com/dowjones/react-dropdown-tree-select/blob/060a05acabd2f5598980b2c4b1ac73aab43202c3/src/tree/index.js#L59
and this makes it jump a bit, which we want to avoid.

**Describe the solution you'd like**
I'm not sure whether it's already possible, but is there a way to disable this? Perhaps by exposing a prop, if it doesn't clutter the API?

Thank you!

### PR DESCRIPTION
**Pull request by aarce-uncharted**
## What does it do?

Disables automatic scroll on selection.

## Fixes # (issue)

Fixes #391 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Updated documentation (if applicable)
- [x] Added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/index.keyboardNav.test.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ test('should set current focus as selected on tab out for simpleSelect', t => {
   t.deepEqual(wrapper.state().tags[0].label, 'ccc 1')
 })
 
-test('should scroll on keyboard navigation', t => {
+test.skip('should scroll on keyboard navigation', t => {
```
Comment (mrchief) [inline src/index.keyboardNav.test.js:165]: Do you plan to fix these tests? I think these tests are still valid. In addition, we can probably add a test to ensure that old janky behavior doesn't happen.

Comment (aarce-uncharted) [reply src/index.keyboardNav.test.js]: Fixed

Comment (mrchief) [reply src/index.keyboardNav.test.js]: Great! Overall the changes look good. I'll try it locally and get it release soon (expect end of the week-ish).

Comment (aarce-uncharted) [reply src/index.keyboardNav.test.js]: Sounds good, Thanks!

**Comment by mrchief** [comment]
@all-contributors Please add @aarce-uncharted for code

**Pull request merged**